spec/helpers/session.rb

```diff
@@ -0,0 +1,20 @@
+module SessionHelpers 
+
+ def sign_in(user)
+    visit'/sessions/new'
```

Comment:
Put space between the method name and the first argument.<br>Use 2 (not 3) spaces for indentation.